selftests/unit/test_utils_service.py

```diff
@@ -190,9 +190,9 @@ def test_start(self):
 
     def test_list(self):
         list_result_mock = mock.Mock(exit_status=0,
-                                     stdout="sshd.service enabled\n"
-                                     "vsftpd.service disabled\n"
-                                     "systemd-sysctl.service static\n")
+                                     stdout=b"sshd.service enabled\n"
```

Comment:
If you change the use to `stdout_text` as mentioned before I think all this tests can be keeped untouched.